addons/web/static/src/views/list/list_renderer.xml

```diff
@@ -253,7 +253,7 @@
                         tabindex="-1">
                         <t t-if="!isInvisible">
                             <t t-if="canUseFormatter(column, record)" t-out="getFormattedValue(column, record)"/>
-                            <Field t-else="" name="column.name" record="record" type="column.widget" class="getFieldClass(column)" fieldInfo="column" t-props="getFieldProps(record, column)"/>
+                            <Field t-elif="column.widget !== 'handle' or !props.readonly" name="column.name" record="record" type="column.widget" class="getFieldClass(column)" fieldInfo="column" t-props="getFieldProps(record, column)"/>
```

Comment:
is it necessary. What about not calling useSortable in the setup if readonly?

Reply:
It is necessary if we don't want to see the handles. useSortable is already disabled by canResequenceRows which is set to false when in readonly

Reply:
Shouldn't handle field itself not render anything? At first I would have said that the handles should be like disabled, but I agree it's better not to have them at all. Maybe the handle field should not render anything in readonly. What do you think?